apps/backoffice-v2/src/components/organisms/Header/Header.Navbar.tsx

```diff
@@ -10,21 +12,42 @@ import { navItems } from '../../../App/nav-items';
  * @constructor
  */
 export const Navbar: FunctionComponent = () => {
+  const { data: filters } = useFiltersQuery();
+  const search = useSearch({
+    strict: false,
+  });
+
   return (
     <nav>
       {navItems.map(({ text, key, children }) => (
         <ul className={`menu menu-compact w-full space-y-2`} key={key}>
           {children?.length > 0 ? (
             <>
-              <div className={`divider`}></div>
               <li className={`menu-title`}>
                 <span className={`gap-x-2`}>{text}</span>
               </li>
-              {children?.map(({ text, href, key }) => (
-                <NavItem href={href} key={key}>
-                  {text}
-                </NavItem>
-              ))}
+              {text === 'Case Management'
+                ? filters?.map(({ id, kind }) => (
+                    <NavItem
+                      href={'/$locale/case-management/individuals'}
```

Comment:
I guess this is temp? if so, try to add TODO: with the change. it will make it easy to review and make sure its not forgotten